tests/dbgp/dbgpclient.php

```diff
@@ -60,8 +93,7 @@ function doRead( $conn )
 			$read = preg_replace( '@(2002-20[0-9]{2})@', '2002-2099', $read );
 			echo $read, "\n\n";
 
-			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) )
-			{
+			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) ) {
```

Comment:
So you wan't me to change back the PHP file to the CS that is was and follow that standard? No problems, should not take too long.